dev-requirements.txt

```diff
@@ -0,0 +1,3 @@
+-e git+https://github.com/ckan/ckan#egg=ckan
+-r https://raw.githubusercontent.com/ckan/ckan/master/requirements.txt
+slackclient==1.0.9
```

Comment:
Perhaps `-r requirements.txt` so you'd be keeping both matching?

Reply:
@DavidLemayian Your question is not clear. Could you please expound?

Reply:
@WNjihia Oops.. Meant instead of having `slackclient==1.0.9` or any other requirement for this extension, we should do `-r requirements.txt` instead to meet these requirements. That way we don't have to define the project's requirements twice. Make sense?

Reply:
Had to read my own comment about five times before making sense of it myself. Sorry about that. 😂

Reply:
@DavidLemayian it's okay. It makes sense. I will make the necessary changes

Reply:
@DavidLemayian This is done.